tests/rptest/services/templates/omb_workload.yaml

```diff
@@ -1,5 +1,8 @@
 name: {{name}}
 
+{% if existing_topic_list is defined %}
+existingTopicList: {{existing_topic_list}}
+{% endif %}
```

Comment:
nitpick: should this be in the next commit?

Reply:
Eh, it should probably be in its own commit. Will refactor it.